Dockerfile.test

```diff
@@ -20,7 +20,7 @@ RUN git clone https://github.com/datajoint/workflow-array-ephys.git .
 # COPY --chown=anaconda:anaconda ./element-array-ephys /main/element-array-ephys
 # RUN pip install /main/element-array-ephys
 # COPY --chown=anaconda:anaconda ./workflow-array-ephys /main/workflow-array-ephys
-# RUN rm /main/workflow-array-ephys/dj_local_conf.json
+# RUN rm -f /main/workflow-array-ephys/dj_local_conf.json
```

Comment:
Does user anaconda have rm permissions in all directories? When I open a CLI for `datajoint/djlab:py3.8-debian`, I don't have permission for `rm` or `mv`